.travis.yml

```diff
@@ -63,8 +63,10 @@ script:
   - PYTHONWARNINGS='module::DeprecationWarning' nosetests -a '!gpu,!slow' --with-doctest tests/install_tests
   - cd tests
   - PYTHONWARNINGS='module::DeprecationWarning' nosetests -a '!gpu,!slow' --with-doctest chainer_tests
-  - cd ..
-  - READTHEDOCS=True python setup.py develop
+  - if [[ $TRAVIS_OS_NAME == "linux" ]]; then
```

Comment:
If this line was executed, some status was outputted.
Please see https://travis-ci.org/pfnet/chainer/jobs/213242912#L877
Is it not enough?